python/test/unit_distributed/test_contexts_arbmpi.py

```diff
@@ -18,12 +18,13 @@
 
 # check Arbor's configuration of mpi
 dict = arb.config()
+config_mpi = dict["mpi"]
 mpi_check = dict["mpi"]
 
 """
 all tests for distributed arb.context using arbor mpi wrappers
 """
-@unittest.skipIf(mpi_check == False, "MPI not enabled!")
+@unittest.skipIf(config_mpi == False, "MPI not enabled!")
```

Comment:
We don't need the exclamation at the end of the string, i.e. just "MPI not enabled" is enough.